.travis.yml

```diff
@@ -90,18 +90,28 @@ jobs:
       if: branch in (master, develop)
       python: 3.6
       install:
-        - pip install -r requirements/deploy.txt
-        - pip install --editable . # install wam, as needed by GB
+        - sudo docker pull ${DOCKER_SLUG}:latest
       script:
-        # create standalone application
-        - gravitybee --src-dir src --sha file --with-latest --extra-data static --verbose
+        - echo "DOCKER_INSTANCE_NAME:${DOCKER_INSTANCE_NAME}"
+        - echo "DOCKER_SLUG:${DOCKER_SLUG}"
+        - |
+          sudo docker run --detach --privileged \
+            --volume="${TRAVIS_BUILD_DIR}":"${TRAVIS_BUILD_DIR}" \
+            --workdir "${TRAVIS_BUILD_DIR}" \
+            --name ${DOCKER_INSTANCE_NAME} \
+            ${DOCKER_SLUG}:latest \
+            init
+        - sudo docker exec ${DOCKER_INSTANCE_NAME} chmod +x ci/deploy.sh
+        - sudo docker exec ${DOCKER_INSTANCE_NAME} ci/deploy.sh
```

Comment:
Figured out that in YAML the line-continuation backslash doesn't work well unless you're in multi-line mode with the pipe. Guess that makes sense but forgot it or never came across it. It kept failing with `docker: invalid reference format` errors without being in multi-line. Now it's one command per line, as you requested, but one of the commands takes several lines.